saltproc/tests/test_app.py

```diff
@@ -8,27 +8,31 @@
 sys.path.append(os.path.dirname(os.path.dirname(path)))
 # global class object
 directory = os.path.dirname(path)
-input_file = directory + '/test'
+iter_inputfile = directory + '/test'
 main_input = directory + '/test.json'
 dot_input = directory + '/test.dot'
 
 serpent = DepcodeSerpent(exec_path='sss2',
-                         template_path=directory + '/template.inp',
-                         input_path=input_file,
+                         template_inputfile_path=directory + '/template.inp',
+                         iter_inputfile=iter_inputfile,
                          iter_matfile=directory + '/material',
-                         geo_file=None)
+                         geo_files=None)
 
 
 def test_read_main_input():
     app.read_main_input(main_input)
-    assert app.neutron_pop == 50
-    assert app.active_cycles == 20
-    assert app.active_cycles == 20
-    assert app.db_file == directory + '/./db_saltproc.h5'
-    assert app.geo_file == ['347_base.ini']
-    assert app.restart_flag is False
-    np.testing.assert_equal(app.power_hist, [1.250E+9, 1.250E+9])
-    np.testing.assert_equal(app.depl_hist, [5, 10])
+    assert app.depcode_inp['codename'] == "serpent"
+    assert app.depcode_inp['npop'] == 50
+    assert app.depcode_inp['active_cycles'] == 20
+    assert app.depcode_inp['active_cycles'] == 20
```

Comment:
Why is this done twice? Is the second one meant to be inactive cycles?

Reply:
Good catch! yes the second line should be inactive cycles!